### ISSUE TITLE
search is broken

### ISSUE THREAD

**Issue by joshribakoff** [facebook/docusaurus#3293]
## 📚 Documentation

Pasted "docsVersionDropdown" into search box at top right of the docs page & hit enter, nothing happens. Search seems broken?

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?

yes

**Comment by slorber** [comment]
Hi,

Search is not broken, it does not look up inside code blocks, and this string only appears in a codeblock.
https://v2.docusaurus.io/docs/theme-classic/#navbar-docs-version-dropdown

So it's more a docs issue: we should add every options available in code blocks documented outside of code blocks, so that search can find it. Do you want to improve the doc of these items?

---


See also 

https://github.com/facebook/docusaurus/issues/3172#issuecomment-669740260

Algolia says searching in code blocks is likely to degrade the search experience, so I guess we should follow his recommendations. 

@s-pace as you can see this problem is quite common :) if Algolia could search in Docusaurus 2 code blocks that would likely be very helpful. Not sure you can do about it right now, just wanted to let you know about this new issue ;)

**Comment by joshribakoff** [comment]
Actually, yes, search was broken. I could not search at all last night. Today I can search again.

It wasn't even popping up saying "no results found", literally nothing happened just like I said. Maybe Algolia was down & there is insufficient feedback to the end user if it is down?

**Comment by joshribakoff** [comment]
Ah, I see the problem. If you paste a search term that simply has 0 results, you get no feedback. If you type the term in character by character, you at least have a chance to see some results & then see them go away.

Ideally there should be a popup that says "no results found for 'docsVersionDropdown'", right now it just acts like the search is broken.

**Comment by slorber** [comment]
![image](https://user-images.githubusercontent.com/749374/90415743-d4aa8680-e0b1-11ea-8d8f-6a6824a662db.png)

To me it does exactly that, no?

**Comment by s-pace** [comment]
Same as @slorber 

I couldn't reproduce this issue. I can confirm you that the search was up last night.

**Comment by joshribakoff** [comment]
https://youtu.be/5YCfHz_R4j0

video showing the reproduction.

**Comment by slorber** [comment]
@joshribakoff this is docusaurus v1, v1 does not even have docsVersionDropdown feature 🤪

**Comment by s-pace** [comment]
Yes the issue is only on docusaurus v1. There is no "no result" dropdown.

**Comment by joshribakoff** [comment]
@slorber that also explains the "no results" :)

@s-page this seems correct, up to you all whether to backport that or not, just figured I'd make you aware.

**Comment by panvalkar1994** [comment]
@joshribakoff I am finding this issue open. But the functionality of the dropdown is working as required. For "docsVersionDropdown" it is showing me "No results for docsVersionDropdown". What am I missing here?
